projects/common/src/utilities/validators/regex-validator.test.ts

```diff
@@ -0,0 +1,14 @@
+import { isValidRegex } from './regex-validator';
+
+const VALID_REGEX = ['2+2', '(2+2) * 3*9'];
+const INVALID_REGEX = ['\\', '['];
+describe('Regex validator', () => {
+  test('can validate regex lists correctly', () => {
+    VALID_REGEX.forEach(regex => {
+      expect(isValidRegex(regex)).toBeTruthy();
+    });
+    INVALID_REGEX.forEach(regex => {
+      expect(isValidRegex(regex)).toBeFalsy();
+    });
```

Comment:
these are two tests. 

1 set for valid and another for invalid. Split them into two tests.

Reply:
updated